docs/contents/cloud-eventarc.json

```diff
@@ -0,0 +1,15 @@
+{
+    "title": "eventarc",
+    "defaultService": "eventarc\/readme",
+    "services": [
+        {
+            "title": "Overview",
+            "type": "eventarc\/readme"
+        },
+        {
+            "title": "EventarcClient (v1)",
+            "type": "eventarc\/v1\/eventarcclient"
```

Comment:
Why do some entries here include a `Client` and some do not?

Reply:
I need some more context to respond to this.

Reply:
I just noticed with the other JSON files in `docs/contents/`, some of them (like `docs/contents/cloud-network-management.json`) have only one entry for the path to the `readme`, and others have a second entry for a `client`

Reply:
Great call out, thank you. They should have a client documented, I'll take a look at my script to see what went wrong and get them updated first thing tomorrow 👍.